### PR TITLE
Fix jinja pin, https for url, SPDX license, pip check

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

- investigating https://github.com/conda-forge/jupyterlab-feedstock/issues/209

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,3 +47,4 @@ extra:
     - blink1073
     - afshin
     - SylvainCorlay
+    - bollwyvl
```
Comment (bollwyvl) [inline recipe/meta.yaml:50]: :wave:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,31 +10,34 @@ source:
...
     - python >=3.5
-    - setuptools
 
   run:
     - python >=3.5
     - json5
     - jsonschema >=3.0.1
     - notebook >=4.2.0
-    - jinja2 =>2.10
+    - jinja2 >=2.10
```
Comment (jasongrout) [inline recipe/meta.yaml:27]: Is this what caused your issue with installing on Windows?

**Comment by jasongrout** [comment]
Thanks!

**Comment by bollwyvl** [comment]
No idea... Just checking upstreams for issues. Also seeing the condition
with just "notebook" and "python=3.7".

**Comment by jasongrout** [comment]
Well, let's see if this build helps you...

**Pull request merged**